charts/tidb-operator/values.yaml

```diff
@@ -97,6 +97,10 @@ controllerManager:
   # - canary-release=v1
   # - k1==v1
   # - k2!=v2
+  securityContext: {}
+  # runAsUser: 1000
+  # runAsGroup: 2000
+  # fsGroup: 2000
```

Comment:
do we need to add other fields (listed in https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#securitycontext-v1-core) as the example?

Reply:
I'll add comment